feature/experimental/tunnel/otg_tests/tunnel_interface_based_ipv4_gre_encapsulation_test/metadata.textproto

```diff
@@ -0,0 +1,20 @@
+# proto-file: github.com/openconfig/featureprofiles/proto/metadata.proto
+# proto-message: Metadata
+
+uuid:  "10287306-7277-456a-ab43-99f63e991152"
+plan_id:  "TUN-1.3"
+description:  "Interface based IPv4 GRE Encapsulation"
+testbed:  TESTBED_DUT_ATE_4LINKS
+ platform_exceptions: {
+   platform: {
+     vendor: JUNIPER
+     hardware_model: "PTX10008"
+     hardware_model: "PTX10001-36MR"
+     hardware_model: "cptx"
+   }
+   deviations: {
```

Comment:
Please mention the bug ID here for new  deviations.

We need bugs to track new  deviations.